### DISCUSSION TITLE
How to handle scroll stop event in v2?

### DISCUSSION THREAD

**Discussion by krutoo** [KingSora/OverlayScrollbars#519] [Q&A]
Hi, i am trying to migrate from v1 to v2 and dont know how to migrate from `callbacks.scrollStop`

How can i handle scroll stop event?

**Comment by KingSora** [comment]
good day @krutoo :)

`v2` has no scrollStop event anymore. I've done this in an attempt to simplify the api and remove mostly unused features. 

Whats the usecase in your case for the `scrollStop` event?

**Reply by krutoo** [reply]
i used scrollstop to keep track of full scroll to the end

**Comment by npearson72** [comment]
@KingSora I'm curious about this also. I have list component with action buttons that contain a lot of complex logic. Ideally while scrolling I would like to avoid rendering those action buttons. However, once scrolling stops, I'd like to allow the action buttons to render.

How might I track when scrolling stops to do this whilst also avoiding triggering re-renders?
